package.json

```diff
@@ -13,6 +13,7 @@
 	"type": "module",
 	"exports": {
 		"types": "./source/index.d.ts",
+		"browser": "./source/exports.js",
 		"default": "./source/index.js"
```

Comment:
There is also the `browser` top-level field, which is older and predates `exports.browser`. I am not sure which style do you prefer, and whether I am missing some other field?

Reply:
No, this is all we need. The `browser` field is underspecified and legacy.